packages/node-app/src/controllers/giveaway.controller.ts

```diff
@@ -108,6 +111,8 @@ export const createGiveaway = async (req: Request, res: Response) => {
     });
     giveawayId = giveaway._id;
 
+    await scheduleWinnerGeneration(giveaway);
```

Comment:
Can we move the schedule to after we call the contract, just in case the contract fails and the giveaway is not created successfully?